src/modules/modals/ConfirmationModal/ConfirmationModal.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Modaliz from 'react-modaliz';
+
+import block from '../../../helpers/bem-cn';
+
```

Comment:
not really sure, we shouldn't have empty lines here I guess

Reply:
just to visually separate third-party libs imports from internal imports, @rafaeljesus